taskchampion/src/replica.rs

```diff
@@ -636,6 +641,13 @@ mod tests {
         assert_eq!(all_tasks.len(), 2);
         assert_eq!(all_tasks.get(&uuid1).unwrap().get_uuid(), uuid1);
         assert_eq!(all_tasks.get(&uuid2).unwrap().get_uuid(), uuid2);
+
+        let mut all_uuids = rep.all_task_uuids().unwrap();
+        all_uuids.sort();
+        let mut exp_uuids = vec![uuid1, uuid2];
+        exp_uuids.sort();
+        assert_eq!(all_uuids.len(), 2);
+        assert_eq!(all_uuids, exp_uuids);
```

Comment:
Also seems like a questionable mutant:

> taskchampion/src/replica.rs:106:9: replace Replica::all_task_uuids -> Result<Vec> with Ok(vec![Default::default()])

Reply:
This particular miss just indicated to me that `all_task_uuids` wasn't covered by tests at all.